src/sb_pointcloud_processing/nodelet_plugins.xml

```diff
@@ -0,0 +1,19 @@
+<library path="lib/libsb_pointcloud_processing">
+
+    <class name="sb_pointcloud_processing/frame_transform"
```

Comment:
I'll probably remove this as well - since pointcloud transformation is done through one of the passthrough nodelet (z).